src/web/src/asset_DELETE.ecpp

```diff
@@ -43,7 +43,7 @@ bool database_ready;
     // verify server is ready
```

Comment:
I'd suggest to add #include <fty_common_macros.h> as well as for other files, not to rely on some header, but to be able to track which files are using this